<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autonomous Driving Demo</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <layout class="QHBoxLayout" name="display_layout" stretch="2,1">
      <item>
       <layout class="QVBoxLayout" name="visualization_canvas_layout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dashboard_canvas_layout"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="control_layout">
      <item>
       <widget class="QPushButton" name="set_goal_button">
        <property name="text">
         <string>Set Goal(A)</string>
        </property>
        <property name="shortcut">
         <string>A</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_pose_button">
        <property name="text">
         <string>Set Pose(S)</string>
        </property>
        <property name="shortcut">
         <string>S</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Cancel(F)</string>
        </property>
        <property name="shortcut">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
